slack/web/async_slack_response.py

```diff
@@ -168,13 +186,6 @@ def validate(self):
         Raises:
             SlackApiError: The request to the Slack API failed.
         """
-        if self._logger.level <= logging.DEBUG:
```

Comment:
This should have been removed in https://github.com/slackapi/python-slack-sdk/pull/1094